### PR TITLE
add PCB ad schematic files

### PR DESCRIPTION
**Pull request by joel5399**

### CODE REVIEW

File: smt_PCB_files/README.txt
Comment (rursprung) [inline smt_PCB_files/README.txt:1]: please turn this into a markdown file

File: smt_PCB_files/README.txt [hunk 1]
```diff
@@ -0,0 +1,3 @@
+How to use：
+
+At editor, open the document via: Top menu - File - Open - EasyEDA... , and select the json file, then open it at the editor, you can save it into a project.
```
Comment (rursprung) [inline smt_PCB_files/README.txt:3]: what editor? [EasyADA](https://easyeda.com/)? if so, please directly add a link to it

File: smt_PCB_files/json_fiels/smt_PCB.json
Comment (rursprung) [inline smt_PCB_files/json_fiels/smt_PCB.json:1]: i don't think that we need a sub-folder for these two JSON files, they can directly reside in this folder here. (also, there's a typo in the folder name though that'll no longer be relevant if they're moved out of the folder: `json_fiels` => `json_files`)

**Review by rursprung — CHANGES_REQUESTED** [review]
thanks for adding those!

please see the review feedback.
nit-picking: small typo in the commit-msg: "ad" -> "and"

**Comment by joel5399** [comment]
thanks for your review

**Review by rursprung — APPROVED** [review]

**Pull request merged**